src/applications/simple_storage_starknetjs.md

```diff
@@ -117,26 +97,9 @@ $ starkli account deploy ./starkli-wallet/account.json
 ```
 This command wants you to fund the address (given in the instructions below the command) in order to deploy the account on the Starknet Sepolia Testnet. We need testnet Ether on Starknet Sepolia which could be obtained from [this](https://starknet-faucet.vercel.app/) testnet faucet. Once the transaction is confirmed on the faucet page, click ENTER on the command line, and the account will be deployed on Starknet Sepolia! Find your account on the [Voyager Sepolia block explorer](https://sepolia.voyager.online/).
 
-### Declaring & Deploying your Contract:
-Firstly, you need to declare your contract which will create a class on Starknet Sepolia. Note that we will use the Sierra program in `./target/simple_storage_SimpleStorage.contract_class.json`.
-
-**Note:** The command below is written to run in the directory of the Scarb folder.
+### Declaring & Deploying the SimpleStorage Contract
 
-```console
-$ starkli declare --keystore /path/to/starkli-wallet/keystore.json --account /path/to/starkli-wallet/account.json --watch ./target/dev/simple_storage_SimpleStorage.contract_class.json
-```
-
-After this command, the class hash is declared. You should be able to find the hash under the command: 
-```console
-Class hash declared:
-0x05c8c21062a74e3c8f2015311d7431e820a08a6b0a9571422b607429112d2eb4
-```
-
-Now, it's time to deploy the contract. Add the clash hash after `--watch`: 
-```console
-$ starkli deploy --keystore /Users/egeaybars123/keystore/keystore --account /Users/egeaybars123/account/account.json --watch 0x05c8c21062a74e3c8f2015311d7431e820a08a6b0a9571422b607429112d2eb4
-```
-You should now see the address of the deployed contract. Note down the address of the contract; we will use it for interacting with the contract using Starknet-js.
+Please refer to **How To Deploy** in the Getting Started part. Note down the address of your contract, you will need it in Starknet-js in the following part.
```

Comment:
Add an actual link with `[How To Deloy](../getting-started/interacting/how_to_deploy.md)`